packages/kbn-ts-type-check-cli/run_type_check_cli.ts

```diff
@@ -163,7 +166,7 @@ run(
         node scripts/type_check --project packages/kbn-pm/tsconfig.json
     `,
     flags: {
-      string: ['project'],
+      string: ['project', 'generateTraces'],
```

Comment:
typo, the field is used as `generateTrace` in other places

Reply:
thanks! last minute change